src/api/database/compact.rst

```diff
@@ -22,15 +22,15 @@
     Request compaction of the specified database. Compaction compresses the
     disk database file by performing the following operations:
 
-    - Writes a new, optimised, version of the database file, removing any
+    - Writes a new, optimized, version of the database file, removing any
```

Comment:
I thought we were standardising on UK spellings?

Reply:
Are we? I missed that memo. But even so, I'm pretty sure that *-ize* is also acceptable British spelling.

Reply:
https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise,_-ize_(-isation,_-ization)

Reply:
That said, I don't mind reverting that change if that's preferred.

Reply:
It's fine, let it go

Reply:
-ize is not British spelling (we get our -ise from French).